dlc_developer_config.toml

```diff
@@ -12,20 +12,20 @@ neuron_mode = false
 graviton_mode = false
 # Please only set it to True if you are preparing a HABANA related PR
 # Do remember to revert it back to False before merging any PR (including HABANA dedicated PR)
-habana_mode = false
+habana_mode = true
 # Please only set it to True if you are preparing a TRCOMP related PR
 # Do remember to revert it back to False before merging any PR (including TRCOMP dedicated PR)
 trcomp_mode = false
 # Please only set it to True if you are preparing a Benchmark related PR
 # Do remember to revert it back to False before merging any PR (including Benchmark dedicated PR)
-benchmark_mode = false
+benchmark_mode = true
```

Comment:
I think that benchmark mode has run successfully as well, right?

Reply:
yes